File: Python/lt_01_estseq/lt01_estseq_23.py
Vet=[]
for i in range(0,4,1):
    Vet.append(float(input("Insira o "+str(i+1)+" º Valor: ")))
for i in range(0,4,1):
    for j in range(i,4,1):
        if Vet[i]>Vet[j]:
            aux=Vet[i]
            Vet[i]=Vet[j]
            Vet[j]=aux
print("\n",Vet)

File: C/lt_01_estseq/lt01_estseq_23.c
#include <stdio.h>

void bubble(float Vet[], int length) {

  int i, j;
  float aux;
  for (i = 0; i < length; i++) {

    for (j = i + 1; j < length; j++) {

      if (*(Vet + i) > *(Vet + j)) {
        aux = *(Vet + i);
        *(Vet + i) = *(Vet + j);
        *(Vet + j) = aux;
      }
    }
  }
}

int main() {
  int i, length;
  float Vet[4];
  length = (sizeof(Vet) / sizeof(Vet[0]));

  // Input
  for (i = 0; i < length; i++) {
    printf("Insira o %d° Numero: ", i + 1);
    scanf("%f", &Vet[i]);
  }

  // Sort
  bubble(Vet, length);

  // Prints
  printf("\nOs Valores serao em ordem:\n");
  for (i = 0; i < 4; i++) {
    printf("%d° -- %.2f\n", i + 1, *(Vet + i));
  }
  return 0;
}
